src/components/LyricView.tsx

```diff
@@ -130,6 +138,11 @@ const LyricView: React.FC<LyricViewProps> = (props: LyricViewProps) => {
       </Fragment>
     );
   }
+
+  function ToggleNavButtonListener() {
+    // 991 is the exact width that the button starts to overlap the lyric view
+    props.setTooNarrow(window.innerWidth <= 991);
```

Comment:
https://www.freecodecamp.org/news/how-to-update-a-components-prop-in-react-js-oh-yes-it-s-possible-f9d26f1c4c6d/#:~:text=Whether%20you%20declare%20a%20component,are%20never%20to%20be%20updated.

`State`s can update within a class, but `props` do not.

Reply:
I think I'm a little confused. The prop isnt getting updated, the `setTooNarrow` prop is a function used to modify it's parent state. It's how I'm setting the `tooNarrow` state that lives in `SongPage.tsx`